### DISCUSSION TITLE
Is there a date when `aspnet:7.0-jammy-chiseled` will move out of Preview?

### DISCUSSION THREAD

**Discussion by xakpc** [dotnet/dotnet-docker#4456] [Q&A] [answered]
Right now it is in `mcr.microsoft.com/dotnet/nightly/`
I read somewhere that it should be moved at the end of 2022. 
Is something changed or wrong with that image?

**Comment by mthalman** [comment, marked as answer]
See https://github.com/dotnet/dotnet-docker/discussions/4137, specifically https://github.com/dotnet/dotnet-docker/discussions/4137#discussioncomment-5079236
